code/datums/quirks/positive_quirks.dm

```diff
@@ -362,3 +362,221 @@
 
 /datum/quirk/item_quirk/signer/remove()
 	qdel(quirk_holder.GetComponent(/datum/component/sign_language))
+
+/datum/quirk/spacer_born
```

Comment:
this is a shit load of code (200+ lines!) just being appended onto this file i think it's time we start splitting it out (starting now). files are free

Reply:
this is gonna be the fourth time we've moved and renamed quirk files